### PR TITLE
Add Open Data Portal Russia and Remove duma.gov.ru

### PR DESCRIPTION
**Pull request by olekstomek**
Add [Open Data Portal Russia](https://data.gov.ru/?language=en)
Associated with: https://github.com/public-apis/public-apis/pull/1922

- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -650,6 +649,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Open Government, New Zealand](https://www.data.govt.nz/) | New Zealand Government Open Data | No | Yes | Unknown |
 | [Open Government, Peru](https://www.datosabiertos.gob.pe/) | Peru Government Open Data | No | Yes | Unknown |
 | [Open Government, Romania](http://data.gov.ro/) | Romania Government Open Data | No | No | Unknown |
+| [Open Government, Russia](https://data.gov.ru/?language=en) | Open Data Portal Russia | Unknown | No | Unknown |
```
Comment (matheusfelipeog) [inline README.md:652]: - auth value is not enclosed with `backticks`
- Unknown is not a valid Auth option. 

Currently, the only accepted inputs for the Auth field are as follows:

`OAuth` - the API supports OAuth
`apiKey` - the API uses a private key string/token for authentication - try and use the correct parameter
`X-Mashape-Key` - the name of the header which may need to be sent
`User-Agent` - the name of the header to be sent with requests to the API
No - the API requires no authentication to run

This API use apiKey for auth: https://data.gov.ru/pravila-i-rekomendacii

```suggestion
| [Open Government, Russia](https://data.gov.ru/?language=en) | Open Data Portal Russia | `apiKey` | No | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @olekstomek 😄 

Can you resolve this review, please?

**Comment by olekstomek** [comment]
@matheusfelipeog done :)

File: README.md (hunk 1 above)
Comment (olekstomek) [reply README.md]: Thanks for this comment. 👍

**Review by matheusfelipeog — APPROVED** [review]
@olekstomek a big THANKS for your update 😄

**Pull request merged**